An initial proposal for C-Team codex (#35)
Applifting/culture: en/c-team-codex.md

# C-Team Codex

The C-Team in Applifting is a group of dedicated leaders who guide the company's direction and embody our core values. They ensure Applifting runs smoothly, grows sustainably, and stays true to its teal principles. By living the ideals in this codex, the C-Team drives Applifting towards its mission of creating digital products with positive impact on planetary scale.

## Lead with Vision, Not Control

> ### Trust in Teal culture

Reviewer: Here I am missing core teal principles since Trusting in Teal should be the way. IMHO this section should answer what Teal culture IS: Be yourself and encourage others to be themselves, take responsibility for yourself by empowering others to accept their responsibility for themselves - not to take their responsibility on your shoulders! Lastly: listen/look around how the environment is changing and adapt to it.
These are my vaguely described 3 core teal principles, please feel free to rephrase.

Lead by setting a clear direction, not by micromanaging. Applifters should be empowered to make decisions and take responsibility.

Your role is to inspire and to remove obstacles. Instead of managing people, mentor them. Help others find their path to success, be it through advice, knowledge, or resources.

> Value each person's contribution, regardless of title or status. Every voice in the organization is important.

Reviewer: ```suggestion
Value each person's contribution, regardless of title or status. Every voice in the organization should be heard.
```
Because the importance of each contribution is assessed individually and if that voice/opinion is against other values, than it is not important and would slow all down.

## Lead by example

First and foremost, be an example for other Applifters. Live by the Applifter's codex.

### Sincerity in Communication

Always speak the truth and ensure honesty in every interaction, whether with team members, partners, or clients.

### Own Your Failures, Share Your Successes

Be accountable for both your successes and failures. Celebrate wins openly, but don’t shy away from admitting mistakes and learning from them. Vulnerability is strength. Be open to feedback from anyone in the organization, regardless of their role.

### Foster Empathy and Emotional Intelligence

Understanding the perspectives and emotions of others is essential. Lead with emotional intelligence, creating an environment where people feel seen and understood. Actively avoid and discourage toxic behaviors.

### Preserve our Culture of Joy and Fulfillment

Leadership is not only about success but also about joy. It is your role to maintain a space where people enjoy their work. Embrace a fun, informal culture where people can be themselves.

### Be an Ambassador

> Externally, be an ambassador of Applifting's values. Communicate openly with customers and partners, ensuring your actions align with the values of transparency and authenticity.

Reviewer: I would move this after Lead by Example, because it points to the same values but outside of the company, and Lead by Example points inside the company.
